src/QGISStyleParser.ts

```diff
@@ -784,16 +784,19 @@ export class QGISStyleParser implements StyleParser {
   }
 
   getQmlFillSymbolFromSymbolizer(symbolizer: FillSymbolizer): any {
+    const fillOpacity = symbolizer.fillOpacity !== undefined ? symbolizer.fillOpacity : symbolizer.opacity;
+    const outlineOpacity = symbolizer.outlineOpacity !== undefined ? symbolizer.outlineOpacity : symbolizer.opacity;
```

Comment:
Just a note:
Actually this is kind of tricky, as you can combine opacities in several symbolizers.
So `opacity`, `fillOpacity` and `outlineOpacity` might get combined. Its quite easy to multiply this when writing, but it's impossible when parsing it back to geostyler-style.

So i guess its ok to support only precise `fillOpacity` and `outlineOpacity` or `opacity`.

Reply:
Yes, I had to implement a distinction for `outlineOpacitiy` and `fillOpacitity` at this point, but to remain compatible I wanted to keep the original `opacitiy` value in mind if it exists.